packages/html-reporter/src/reportView.css

```diff
@@ -29,6 +29,10 @@ body {
   width: 100%;
 }
 
+.error {
```

Comment:
global style with the name like this would be a bad idea.

Reply:
CSS Modules would solve it - so it would not conflict with other components. Afaik a common best-practise nowadays and [built-in in Vite](https://vite.dev/guide/features#css-modules). Maybe a good time to try?

Reply:
I'd love to move to CSS modules. I assumed in this case that the global `error` would be correct for its use.